docs/manual/manual_stock_led.rst

```diff
@@ -0,0 +1,251 @@
+with your module's GPIO2 and GPIO3 connections where GPIO2 controls the RGB LED and GPIO3 controls the white LEDs. Depending on the size of your module, 
+you will see the GPIO values printed as either IOX or GPIOX where X is the GPIO value.
+
+.. list-table:: LED Connection Examples
+   :class: borderless
+   :align: center
+
+   * - .. figure:: ../_static/manual_images/leds/40_led_connections.png
+            :alt: 40-06 LED Connections
+            :align: center
+            :height: 350
+
+            Vertiq 40-06 LED Connections
+     - .. figure:: ../_static/manual_images/leds/81_led_connections.png
+            :alt: Vertiq 81-08 Gen 2 LED Connections
+            :align: center
+            :height: 350
+
+            Vertiq 81-08 Gen 2 LED Connections
+
+The LED board uses a 4-pin JST-ZE connector in order to interface with your module. The following image displays the LED board's pinout.
+
+<INSERT NICE PICTURE HERE OF THE PINOUT AND OF THE BOARD>
+
+Please note that the connector's clip is at the top of this image. Take care to ensure that the module's RGB LED pad connects with the LED board's RGB LED pin, 
+and the module's white LED pad attaches to the LED board's white LED pin. Failure to do so can result in damage to your LED board as well as your module.
+
+Controlling Your LEDs
+========================
+.. warning::
+    Do not plug in your module's LED board while the module is already powered up. “Hot plugging” your module this way can cause damage to the LED board as well 
+    as the module itself. Always ensure that the LED board is fully connected with the module before powering it on.
+
+Default Pattern
+******************
+
+By default, Vertiq modules control the RGB LED to display a static blue color, and the white LEDs to be shut off. On startup, the RGB LED should be illuminated.
+
+<try to get a nice picture of that>
+
+RGB LED Configuration
+***********************
+Your module has control over the RGB LED's red, green, and blue intensities as well as its strobing output. Strobing configuration is identical for both the RGB and 
+white LEDs, and is covered in :ref:`strobing_configuration`. By default, strobing is disabled for your RGB LED. RGB LED control parameters are found in 
+the :ref:`RGB LED client reference table <rgb_led_message_table>`.
+
+The pertinent control parameters for RGB LED control are ``red``, ``green``, ``blue``, and ``update_color``. Each of the RGB parameters can be set, gotten, and 
+saved, and can be an integer [0, 255] where 255 indicates a maximum intensity. ``update_color`` is a trigger mechanism that causes your module to command the LED 
+with the intensities specified by the values in ``red``, ``green``, and ``blue``.
+
+Suppose you wanted to set your LED to output a bright yellow light for 3 seconds, and then turn red only before turning off completely after another 3 seconds. 
```

Comment:
I had a bit of trouble parsing the goal here at first, may be can be simplified to:

"Suppose you wanted your RGB LED to be bright yellow for 3 seconds, bright red for 3 seconds, and then turn off completely".

I think that captures what the example is saying in a bit more straightforward way